qiskit_algorithms/__init__.py

```diff
@@ -11,29 +11,44 @@
 # that they have been altered from the originals.
 
 """
-=====================================
-Algorithms (:mod:`qiskit_algorithms`)
-=====================================
-It contains a collection of quantum algorithms, for use with quantum computers, to
-carry out research and investigate how to solve problems in different domains on
-near-term quantum devices with short depth circuits.
-
-Algorithms configuration includes the use of :mod:`~qiskit_algorithms.optimizers` which
-were designed to be swappable sub-parts of an algorithm. Any component and may be exchanged for
-a different implementation of the same component type in order to potentially alter the behavior
-and outcome of the algorithm.
+============================================
+Qiskit Algorithms (:mod:`qiskit_algorithms`)
+============================================
+Qiskit Algorithms is a library of quantum algorithms for quantum computing with
+`Qiskit <https://qiskit.org>`_.
+These algorithms can be used to carry out research and investigate how to solve
+problems in different domains on simulators and near-term real quantum devices
+using short depth circuits.
+
+Now there are some classical algorithms here, for example the :class:`.NumPyMinimumEigensolver`,
+which take the same input but solve the problem classically. This has utility in the near-term,
+where problems are still tractable classically, to validate and/or act as a reference.
+There are also classical :mod:`.optimizers` for use with variational algorithms such as :class:`.VQE`.
+
+While :mod:`.optimizers` were been mentioned, algorithms may also use :mod:`.gradients` and
+:mod:`.state_fidelities`. Any of these may also be used directly if you have need of such function.
+
+The quantum algorithms here all use ``Primitives`` to execute quantum circuits. This can be an
+``Estimator``, which computes expectation values, or a ``Sampler`` which computes
+probability distributions. Refer to the specific algorithm for more information in this regard.
```

Comment:
```suggestion
Other common subroutines used by this module's algorithms are :mod:`.gradients` and
:mod:`.state_fidelities`. Any of these may also be used directly if you have need of such function.

The quantum algorithms here all use ``Primitives`` to execute quantum circuits. This can be an
``Estimator``, which computes expectation values, or a ``Sampler`` which computes
probability distributions. Refer to the specific algorithm for more information in this regard.
```